docs/monitoring/README.md

```diff
@@ -0,0 +1,79 @@
+# Prometheus Monitoring for TF operator
+
+## Available Metrics
+
+Currently available metrics to monitor are listed below.
+
+### Metrics for Each Component Container for TF operator
+
+Component Containers:
+* tf-operator
+* tf-master
```

Comment:
This should be tf-chief, to match with the most recent TensorFlow semantics.

Reply:
Making the change